README.md

```diff
@@ -28,6 +28,12 @@ Select a region and screenshoot it:
 grim -g "$(slurp)"
 ```
 
+Select a region and screenshoot it, then edit it using `swappy`:
```

Comment:
I'd rather not add this one, it's already in swappy's README.

Reply:
If people don't know about swappy, then they can't read its README. But I'll remove it if that's your decision.

Reply:
The thing is I recently switched to Wayland and Sway and I'm gradually finding new tools. And when I found grim, I first read its manual, and it did not have any examples. Later when I read grim's README, I thought I should have done this sooner.